### PR TITLE
chore(sdk-api): make web entry work

### PR DESCRIPTION
**Pull request by bitgoAaron**
- hoist webpack dependencies to root package
- move base webpack config to root folder
- define correct entry points for sdk-api package.json
- fix lint errors in unspents
- stop web-packing account-lib

Ticket: BG-50241

### CODE REVIEW

File: modules/statics/package.json [hunk 1]
```diff
@@ -22,8 +22,7 @@
     "directory": "modules/statics"
   },
   "devDependencies": {
-    "nyc": "^15.0.0",
-    "should": "^13.2.3"
+    "nyc": "^15.0.0"
```
Comment (tylerlevine) [inline modules/statics/package.json:25]: nit: could remove nyc here

File: modules/sdk-test/package.json [hunk 2]
```diff
@@ -33,7 +33,6 @@
     "@bitgo/statics": "^6.17.0",
     "bignumber.js": "^8.0.1",
     "lodash": "^4.17.14",
-    "should": "^13.1.3",
     "should-http": "^0.1.1"
```
Comment (tylerlevine) [inline modules/sdk-test/package.json:36]: nit: do we need `should-http` if we don't have `should`?

**Review by tylerlevine — APPROVED** [review]
small nits. Awesome work on this 🙏

File: modules/statics/package.json (hunk 1 above)
Comment (bitgoAaron) [reply modules/statics/package.json:25]: There's quite a few unused packages throughout the repo. I'd like to address all of them at once in this ticket:
https://bitgoinc.atlassian.net/browse/BG-51022

File: modules/sdk-test/package.json (hunk 2 above)
Comment (bitgoAaron) [reply modules/sdk-test/package.json:36]: `should` is provided by the root package.

**Pull request merged**